Makefile

```diff
@@ -82,121 +15,21 @@ ci-shell: clean .dapper                  ## Launch a shell in the CI environment
-	docker run --rm --link ${PROG}-dev-shell:${PROG}-server --name ${PROG}-peer${PEER} --hostname ${PROG}-peer${PEER} -p 127.0.0.1:234${PEER}:2345 -ti -e WORKSPACE=$$(pwd) -v $${HOME}:$${HOME} -v ${PROG} -w $$(pwd) --privileged -v ${PROG}-pkg:/go/pkg -v ${PROG}-cache:/root/.cache/go-build ${PROG}-dev bash
-
-dev-peer-enter:                         ## Enter the peer shell on another terminal
-	docker exec -it ${PROG}-peer${PEER} bash
-
-artifacts: build download-charts package-airgap
-	cp bin/${PROG} dist/artifacts/${RELEASE}
-
-.ci: validate-ci artifacts
+.ci: validate build package
 
 in-docker-%: .dapper                     ## Advanced: wraps any target in Docker environment, for example: in-docker-build-debug
 	mkdir -p ./bin/ ./dist/
 	./.dapper -f Dockerfile --target dapper make $*
 
-CHARTS_DIR = build/static/charts
-MANIFEST_DIR = manifests
-CHARTS = canal:v3.13.3 coredns:1.10.101 kube-proxy:v1.18.4 metrics-server:2.11.100 nginx-ingress:1.36.300
-download-charts:
-	mkdir -p $(CHARTS_DIR)
-	for chart in $(CHARTS); do \
-    	  chart_name=`echo "$${chart}" | cut -d ":" -f 1`;\
-    	  chart_version=`echo "$${chart}" | cut -d ":" -f 2`;\
-    	  curl -sfL https://dev-charts.rancher.io/$$chart_name/$$chart_name-$$chart_version.tgz -o ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz;\
-    	  chart_content=`base64 -w 0 ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz`;\
-    	  sed -e "s|%{CHART_CONTENT}%|$$chart_content|g" $(MANIFEST_DIR)/$$chart_name.yml > $(CHARTS_DIR)/$$chart_name-chart.yml;\
-    	  rm ${CHARTS_DIR}/$$chart_name-$$chart_version.tgz;\
-    done
-
 ./.dapper:
 	@echo Downloading dapper
 	@curl -sL https://releases.rancher.com/dapper/v0.5.0/dapper-$$(uname -s)-$$(uname -m) > .dapper.tmp
 	@@chmod +x .dapper.tmp
 	@./.dapper.tmp -v
 	@mv .dapper.tmp .dapper
 
-k8s-image: k8s-image-build k8s-image-scan
-
-k8s-image-build:
-	docker build \
-    	--build-arg TAG=${DRONE_TAG} -f Dockerfile.k8s -t ranchertest/kubernetes:${VERSION}-${GOARCH} .
-
-SEVERITIES = HIGH,CRITICAL
-k8s-image-scan:
-	trivy --severity $(SEVERITIES) --no-progress --skip-update --ignore-unfixed ranchertest/kubernetes:${VERSION}-${GOARCH}
-
-k8s-image-publish: k8s-image
-	docker push ranchertest/kubernetes:${VERSION}-${GOARCH}
-
-# currently works only with amd64
-image-manifest:
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create --amend ${REPO}/rke2-runtime:${VERSION} ${REPO}/rke2-runtime:${VERSION}-${GOARCH}
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push ${REPO}/rke2-runtime:${VERSION}
-
-dispatch:
-	curl -XPOST -u "${PAT_USERNAME}:${PAT_TOKEN}" \
-            -H "Accept: application/vnd.github.everest-preview+json"  \
-            -H "Content-Type: application/json" https://api.github.com/repos/rancher/rke2-upgrade/dispatches \
-            --data '{"event_type": "create_tag", "client_payload": {"tag":"'"${DRONE_TAG}"'"}}'
-
-package-airgap:
-	mkdir -p dist/artifacts
-	scripts/package-airgap
+$(TARGETS):
```

Comment:
Might want to declare as phony

Reply:
PHONY is required, for example `build` will conflict with `build/`